hugegraph-core/src/main/java/com/baidu/hugegraph/backend/tx/IndexableTransaction.java

```diff
@@ -57,6 +57,15 @@ protected void commit2Backend() {
         this.commitMutation2Backend(mutation, txMutation);
     }
 
+    @Override
+    public void commitIfGtSize(int size) throws BackendException {
+        int totalSize =
+            this.mutationSize() + this.indexTransaction().mutationSize();
```

Comment:
int totalSize = this.mutationSize();
totalSize += this.indexTransaction().mutationSize();